integration-tests/main-yaml/src/main/resources/application.properties

```diff
@@ -0,0 +1,21 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+
+#
+# Main
+#
+camel.main.routes-include-pattern = classpath:routes/my-routes.yaml,classpath:routes/my-rests.yaml
```

Comment:
Don't we want to coin some convention to include some pattern by default? `routes/**.yaml` maybe?

Reply:
wildcard does not seem to work with native compilation so we either need to change the way we bundle classpath resources to native image or we have to use full paths

Reply:
Oh, sorry, I have misread it as native resource inclusion. You are right, that wildcards neither work here nor would be easy to  implement.